src/main/java/seedu/address/storage/RequiredCommandMessages.java

```diff
@@ -19,9 +16,14 @@
     public static final String MESSAGE_MATHANDSCI = "These are the Math and Science Modules you have yet to take:";
     public static final String MESSAGE_SUCCESS_MATHANDSCI = "You have completed your "
             + "Math and Science Module Requirement!";
-    public static final Path FOUNDATION_PATH = Paths.get("src/main/resources/data/foundationmodules.json");
-    public static final Path INTERNSHIP_PATH = Paths.get("src/main/resources/data/industrialexperience.json");
-    public static final Path ITPROF_PATH = Paths.get("src/main/resources/data/ITProfessionalism.json");
-    public static final Path MATHANDSCI_PATH = Paths.get("src/main/resources/data/mathandsciencemodules.json");
-    public static final Path SCIENCE_PATH = Paths.get("src/main/resources/data/sciencemodules.json");
+    public static final String FOUNDATION_PATH = "data/foundationmodules.json";
+    public static final String INTERNSHIP_PATH = "data/industrialexperience.json";
+    public static final String ITPROF_PATH = "data/ITProfessionalism.json";
+    public static final String MATHANDSCI_PATH = "data/mathandsciencemodules.json";
+    public static final String SCIENCE_PATH = "data/sciencemodules.json";
+    public static final String TEST_FOUNDATION_PATH = "src/main/resources/data/foundationmodules.json";
+    public static final String TEST_INTERNSHIP_PATH = "src/main/resources/data/industrialexperience.json";
+    public static final String TEST_ITPROF_PATH = "src/main/resources/data/ITProfessionalism.json";
+    public static final String TEST_MATHANDSCI_PATH = "src/main/resources/data/mathandsciencemodules.json";
+    public static final String TEST_SCIENCE_PATH = "src/main/resources/data/sciencemodules.json";
```

Comment:
Great point, I've shifted it to the `RequiredCommandStorageTest` class and the other test classes now import from there instead of the `RequiredCommandMessages` class.